src/support.sch

```diff
@@ -16,10 +17,40 @@
         <rule context="*">
             <assert test="
                 name()='article'
+                or name()='kwd'
```

Comment:
`<front>` and `<article-meta>` will be added in #2.

Reply:
We will ignore `<kwd-group>` outside of `<article-meta>` (eg `<media>`). Currently this is the case as they're all ignored themselves, but the `<kwd-group>` there won't actually be flagged as ignored.

Reply:
(Same with #3 in the `<body>` only.)

Reply:
Added in cb740e3.